### PR TITLE
Bump isomorphic-fetch to v3.0.0

### PR DESCRIPTION
**Pull request by eddieferrer**
* Bump dependency
* Fix deprecated headers.getAll()

### CODE REVIEW

File: server/util/getSessionCookies.js [hunk 1]
```diff
@@ -23,15 +23,16 @@ module.exports = function getSessionCookies(url = '', requestCookies = {}) {
 					Cookie: getCookieString(requestCookies),
 				},
 			}).then(res => {
-				const setCookies = res.headers.getAll('set-cookie');
-				const parsed = setCookieParser(setCookies, { decodeValues: false });
+				const combinedCookieHeader = res.headers.get('set-cookie');
```
Comment (eddieferrer) [inline server/util/getSessionCookies.js:26]: This returns a comma separated list, where getAll() returned an array

File: server/util/getSessionCookies.js [hunk 2]
```diff
@@ -23,15 +23,16 @@ module.exports = function getSessionCookies(url = '', requestCookies = {}) {
...
+				const splitCookieHeaders = setCookieParser.splitCookiesString(combinedCookieHeader);
```
Comment (eddieferrer) [inline server/util/getSessionCookies.js:27]: Turns comma list into array

**Comment by eddieferrer** [comment]
Used this documentation example: https://www.npmjs.com/package/set-cookie-parser#usage-in-react-native
Seems to be working for me, definitely not crashing with an error anymore. That being said, I couldn't produce a test case locally where `set-cookie` was actually part of the response header

File: server/util/getSessionCookies.js (hunk 1 above)
Comment (eddieferrer) [inline server/util/getSessionCookies.js:26]: I couldn't reproduce a case locally where these headers are present

**Review by emuvente — APPROVED** [review]
This looks correct to me. I'll try it out locally to see if I can get the expected headers to show up.

**Comment by emuvente** [comment]
Testing locally worked as expected. Also it looks like the failing travis test is just a transient error. You could push to this branch again to restart the build to confirm, but I think this is good to merge

**Review by mcstover — APPROVED** [review]

**Pull request merged**